client.lua

```diff
@@ -29,6 +29,14 @@ local function SplitStr(inputstr, sep)
 end
 
 local function connecttoradio(channel)
+    if channel > Config.MaxFrequency or channel <= 0 then QBCore.Functions.Notify(Lang:t('restricted_channel_error'), 'error') return false end
+    if Config.RestrictedChannels[channel] ~= nil then
+        local isJobWhitelisted = Config.RestrictedChannels[channel][PlayerData.job.name]
+        if (not isJobWhitelisted) or (isJobWhitelisted and not PlayerData.job.onduty) then
```

Comment:
Refer to my comment on the previous thread about changing these conditions.

Reply:
You're tottally correct, what an stupid condition I did jajaja, being corrected right now

Reply:
No worries lol, easily done. Thank you for your changes once again.